### ISSUE TITLE
Plotting Blocks Grading

### ISSUE THREAD

**Issue by wesharper** [webfun-sept2017/Karina-Naumenko-webfun-sept2017#1]
advertisement should be right aligned with last .subcontents div.
https://github.com/webfun-sept2017/Karina-Naumenko-webfun-sept2017/blob/524af1aa1fc51c4dc970c6bbd0061736f5ee60a7/CSS/style.css#L42-L47

**Comment by karinanaumenko** [comment]
I know it is very painful to look through someone else's code, and I thank you for grading my assignment, but your comment gives me zero insight for understanding my mistake if I made one.

**Comment by nguyenhmp** [comment]
Hey karina, this was just a tiny flaw that your purple box(#advertisement) was not completely lined up with the last yellow one(.subcontents) if that makes sense. But other than that, it looks fine. Sorry for the confusion!

**Comment by wesharper** [comment]
Yes, this was very nitpicky. It didn't exactly match the image from the assignment's example, and it was easy for me to spot at a glance.
